tests/framework/extensions/rancherversion/rancherversion.go

```diff
@@ -0,0 +1,31 @@
+package rancherversion
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+)
+
+/* Requests the rancher version from the rancher server, parses the returned
+ * json and returns a Struct object, or an error.
+ */
+func RequestRancherVersion(rancher_url string) (Config, error) {
```

Comment:
Return the pointer Caleb, so you don't have dereference at the Botton on line 30